certbot-ci/setup.py

```diff
@@ -11,6 +11,7 @@
     'pytest',
     'pytest-cov',
     'pytest-xdist',
+    'python-dateutil',
```

Comment:
We should pin this in `tools/dev_constraints.txt`.

Reply:
It is already, on master: https://github.com/certbot/certbot/blob/56f609d4f582f61274f93a807ec4b8a0bba43828/tools/dev_constraints.txt#L62

Reply:
Indeed it is! Sorry for the noise.